rabies/analysis_pkg/cpca/modeling.py

```
@@ -0,0 +1,203 @@
+    
+    C = closed_form(W, X).T
+        
+    Cnet = C[:,:n_prior]
+    Cs = C[:,n_prior:n_prior+n_s]
+    Ct = C[:,n_prior+n_s:]
+
+        
+    return Cnet,Wnet,Cs,Ws,Ct,Wt,C,W
+
+
+def cpca_quick(X, C_prior, sequence = ['s','s','s','s'], tol=1e-10, verbose=False):
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)